main.js

```diff
@@ -68,6 +79,71 @@ function main(args) {
     }
 }
 
+function createStatsClient(val) {
+    var opts = null;
+    var matchesHostPort = val.match(/^(.+):(\d+)$/);
```

Comment:
I believe this approach to parse this string is quite brittle; see my comment on https://github.com/uber/ringpop-go/pull/144/files#r64207110.